### PR TITLE
Issue #57: change docroot 

### PR DESCRIPTION
**Pull request by cyberrodent**
re Issue #57 
- Creates a htdocs directory. Move index.php and assets inside.
- Update places where we require and include files.
- Update example vhost and php -S invocation in README
- replace error_log with $log->error (and debug)

### CODE REVIEW

File: www/index.php [hunk 1]
```diff
@@ -14,9 +14,14 @@
 
 $config = Configuration::get_configuration();
 $app = new Slim();
+$app->config('debug', true);
```
Comment (mrtazz) [inline www/index.php]: should we wrap this into something like `if (getenv('MORGUE_ENVIRONMENT') === "development")`?

File: www/index.php [hunk 2]
```diff
@@ -94,9 +99,9 @@ function default_status_time() {
  */
 foreach ($config['feature'] as $feature) {
     if ($feature['enabled'] == "on") {
-        error_log("Including Feature {$feature['name']}");
-        include $feature['name'] . '/lib.php';
-        include $feature['name'] . '/routes.php';
+        $log->debug("Including Feature {$feature['name']}");
```
Comment (mrtazz) [inline www/index.php]: probably also worth wrapping into a dev env check

**Comment by mrtazz** [comment]
I like this change. Should we call it `htdocs` instead of `www`?

File: www/index.php (hunk 1 above)
Comment (cyberrodent) [inline www/index.php]: I think just remove it.  I don't see a strong need to be opinionated about if you run debug mode or not in dev.  This did spark half a thought as to not running debug mode in production, but again, I don't think we need to express an opinion about that here.

File: www/index.php (hunk 2 above)
Comment (cyberrodent) [inline www/index.php]: It seems the log levels help here.  We can set the log level when/right after initializing $app.  Something like ```$log->setLevel(\Slim\Log::DEBUG);```  I can add this as well

**Comment by cyberrodent** [comment]
I like ```htdocs``` too.

File: www/index.php (hunk 2 above)
Comment (cyberrodent) [inline www/index.php]: We're a bit behind the docs online. I found this helpful 
https://github.com/etsy/morgue/blob/master/vendor/slim/slim/docs/logging.markdown

**Pull request merged**